Neos.Neos/Resources/Private/Schema/NodeTypes.schema.yaml

```diff
@@ -107,6 +107,10 @@ additionalProperties:
 
                       'placeholder': { type: string, description: "Optional text that will be rendered in the aloha editor when there is no content." }
 
+                      'sampleCase' : { type: any }
+
+                      'fallbackCase' :  { type: any }
```

Comment:
right, but we should probably get rid of those weird tests or at least adjust them to something meaningful instead.. adding this to the schema just causes confusion